lang/main.json

```diff
@@ -428,5 +428,13 @@
         "noOtherDevices": "No other devices available",
         "selectADevice": "Select a device",
         "testAudio": "Test sound"
+    },
+    "invite": {
+        "addPassword": "Add password",
+        "hidePassword": "Hide password",
```

Comment:
I took a pass just took another pass and removed a couple strings related to room locking that weren't being used. createPassword is being used as the placeholder in the input for adding a password.